mlflow_faculty/trackingstore.py

```diff
@@ -292,5 +273,38 @@ def search_runs(self, experiment_ids, search_expressions, run_view_type):
         else:
             return [faculty_run_to_mlflow_run(run) for run in faculty_runs]
 
-    def log_batch(self, run_id, metrics, params, tags):
-        raise NotImplementedError()
+    def log_batch(self, run_uuid, metrics=None, params=None, tags=None):
+        """
+        Fetches the experiment by ID from the backend store.
+
+        :param experiment_id: Integer id for the experiment
+        :param run_uuid: string containing run UUID
+            (32 hex characters = a uuid4 stripped off of dashes
+        :param metrics: List of Mlflow Metric entities.
+        :param params: List of Mlflow Param entities
+        :param tags: List of Mlflow Tag entities.
+        """
+        metrics = set_to_empty_list_none(metrics)
```

Comment:
```suggestion
        metrics = [] if metrics is None else list(metrics)
```

Reply:
That's better than I thought it would look. Will do

Reply:
I implemented this was unable to remove the list enclosing `map()` sine in python3 this returns a map object